crates/krun/src/bin/krun.rs

```diff
@@ -115,6 +117,24 @@ fn main() -> Result<()> {
         }
     }
 
+    if let Ok(run_path) = env::var("XDG_RUNTIME_DIR") {
+        let pulse_path = Path::new(&run_path).join("pulse/native");
+        if pulse_path.exists() {
+            let c_pulse_path = CString::new(
```

Comment:
Just shadow the name `pulse_path` since we don't need the `PathBuf` anymore?

```suggestion
            let pulse_path = CString::new(
```